packages/sdk/src/ValidationRules.ts

```diff
@@ -31,28 +36,38 @@ export class ValidationRules {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-validation-rules-assignment
 	 */
-	public createAssignment(validationRuleId: string, assignment: $FixMe) {
-		return this.client.post(`/validation-rules/${encode(validationRuleId)}/assignments`, assignment)
+	public createAssignment(validationRuleId: string, assignment: T.ValidationRulesCreateAssigment) {
+		return this.client.post<T.ValidationRulesCreateAssigmentResponse>(
+			`/validation-rules/${encode(validationRuleId)}/assignments`,
+			assignment,
+		)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-validation-rules-assignment
 	 */
 	public deleteAssignment(validationRuleId: string, assignmentId: string) {
 		return this.client.delete(`/validation-rules/${encode(validationRuleId)}/assignments/${encode(assignmentId)}`)
 	}
+
+	// @todo - I can't find any information regarding this method
+
 	public validate(validationRuleId: string, params?: $FixMe) {
```

Comment:
Use `params = {}`

Reply:
The type is the same as the one for voucher validation so let's do it later once voucher types PR is mergd